src/compute/src/rpc/service/config_service.rs

```diff
@@ -32,9 +33,9 @@ impl ConfigService for ConfigServiceImpl {
         _request: Request<ShowConfigRequest>,
     ) -> Result<Response<ShowConfigResponse>, Status> {
         let batch_config = serde_json::to_string(self.batch_mgr.config())
-            .map_err(|e| tonic::Status::internal(format!("{}", e)))?;
+            .map_err(|e| e.to_status(Code::Internal, "compute"))?;
         let stream_config = serde_json::to_string(&self.stream_mgr.config().await)
-            .map_err(|e| tonic::Status::internal(format!("{}", e)))?;
+            .map_err(|e| e.to_status(Code::Internal, "compute"))?;
```

Comment:
Use `to_status` to convert an error to `tonic::Status`. If this is not ad-hoc, you may want to directly `impl From<..> for tonic::Status`.